exercises/practice/anagram/.meta/config.json

```diff
@@ -1,6 +1,16 @@
 {
   "blurb": "Given a word and a list of possible anagrams, select the correct sublist.",
   "authors": [],
```

Comment:
seems like the author is not on github anymore.

https://github.com/exercism/csharp/commit/4d73a89429b3eda350c3107c4f36091e792284a6#diff-f728d1403cb3fe678829db17dc5f51008197dc4d22c88984a1af842e80b6c206